samples/src/main/fabric/NYC-GreenTaxi-Read-Write-Data-To-Kusto.ipynb

```diff
@@ -0,0 +1,168 @@
+    "kustoUri = \"https://<fabric-kusto>.kusto.data.microsoft.com\"#e.g.\"https://fabrickusto.z0.kusto.data.microsoft.com\"\n",
+    "database=\"Stocks\"\n",
+    "table=\"GreenTaxiData\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "cellStatus": "{\"Ramachandran G\":{\"queued_time\":\"2023-03-23T08:40:46.5885086Z\",\"session_start_time\":null,\"execution_start_time\":\"2023-03-23T08:40:46.965185Z\",\"execution_finish_time\":\"2023-03-23T08:49:19.0524624Z\",\"state\":\"finished\",\"livy_statement_state\":\"available\"}}",
+    "jupyter": {
+     "outputs_hidden": false,
+     "source_hidden": false
+    },
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "outputs": [],
+   "source": [
+    "#This is an example of writing data to Kusto. The source data is read as a blob into a dataframe from Azure Open Data for GreenTaxi / Limousines in NYC.\n",
+    "#The access token is created using the user's credential and will be used to write the data to the Kusto table GreenTaxiData, therefore the user is required \n",
+    "#for 'user' privileges or above on the target database and table 'admin' privileges if the table already exists. If the table does not exist, \n",
+    "#it will be created with the DataFrame schema.\n",
+    "df.write.format(\"com.microsoft.kusto.spark.synapse.datasource\").\\\n",
+    "option(\"kustoCluster\",kustoUri).\\\n",
+    "option(\"kustoDatabase\",database).\\\n",
+    "option(\"kustoTable\", table).\\\n",
+    "option(\"accessToken\", mssparkutils.credentials.getToken(kustoUri)).\\\n",
+    "option(\"tableCreateOptions\", \"CreateIfNotExist\").mode(\"Append\").save()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "cellStatus": "{\"Ramachandran G\":{\"queued_time\":\"2023-03-23T08:49:29.7202163Z\",\"session_start_time\":null,\"execution_start_time\":\"2023-03-23T08:49:30.1748177Z\",\"execution_finish_time\":\"2023-03-23T08:49:30.6119991Z\",\"state\":\"finished\",\"livy_statement_state\":\"available\"}}",
+    "jupyter": {
+     "outputs_hidden": false,
+     "source_hidden": false
+    },
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "outputs": [],
+   "source": [
+    "#This is an example of Reading data from Kusto. Here the query retrieves the max,min fares and distances that the taxi recorded every month from the years 2014 to 2020\n",
+    "kustoQuery = \"GreenTaxiData |  where puYear between (2014 .. 2020 ) | summarize  MaxDistance=max(tripDistance) , MaxFare = max(fareAmount) ,MinDistance=min(tripDistance) , MinFare = min(fareAmount) by puYear,puMonth | order by puYear,puMonth desc\"\n",
+    "kustoDf  = spark.read\\\n",
+    "            .format(\"com.microsoft.kusto.spark.synapse.datasource\")\\\n",
+    "            .option(\"accessToken\", mssparkutils.credentials.getToken(kustoUri))\\\n",
```

Comment:
should we change it to 'kusto'  and test this once  ?

Reply:
sure